build/regressions.config.yml

```diff
@@ -90,7 +90,7 @@ sources:
   - name: PublishedSizeAot
     table: NativeAotBenchmarks
     rules:
-      - include: Stage1Aot
+      - include: Stage*Aot
```

Comment:
What does `*` mean? ;)

Reply:
Same thing it means here:

https://github.com/aspnet/Benchmarks/blob/fc052c63779c4faa17cd5bbc13c79509a6b6ad3d/build/regressions.config.yml#L53

Reply:
Exactly, so I don't think that's what you want. This field is a regular expression. Other example: `ulib|fasthttp|actix|gin|express|node|netty`

Reply:
Does this not work, like a normal person would expect?

Reply:
oh, so it isn't a wildcard. but a regex. I need a dot.

Reply:
`Stage.Aot`, `Stage(1|2)Aot`, `Stage1Aot|Stage2Aot`, `Stage\dAot`, `Stage.*Aot` ...

Reply:
Better?

Reply:
my suggestions won't work with grpc obviously, bad me